src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Resource/MoveAzureResourceCmdlet.cs

```diff
@@ -67,6 +61,12 @@ public class MoveAzureResourceCommand : ResourceManagerCmdletBase
         public string[] ResourceId { get; set; }
 
         /// <summary>
+        /// Gets or sets a value that indicates if the user should be prompted for confirmation.
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
```

Comment:
The order of the parameters were changed. This was to fix an auto-populating issue on parameters.